04_HowTos/QueryClient/How_to_query_DESI_DR1_Data.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Could this question be accomplished by using the <code>lastnight</code>  column instead of the complex query below? Or maybe that could be an independent check afterwards. This is also an optional suggestion and could be left for future versions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Possibly, but this notebook is intended to replicate the queries in the database introduciton DESI tutorial, so it would require changes in multiple places. Definitely something to consider after DR1.</p>